requirements.txt

```diff
@@ -14,6 +14,7 @@ pyflakes==1.6.0
 pylint==1.8.2
 python-dateutil==2.7.0
 python-slugify==1.2.4
+git+https://github.com/beelit94/python-terraform.git@develop#egg=python-terraform
```

Comment:
The stablest Python library for Terraform I could find, will pin it to a commit or to the latest release when `0.10.0` is out.